packages/appium/lib/appium.js

```diff
@@ -195,6 +195,16 @@ class AppiumDriver extends DriverCore {
     }));
   }
 
+  async getAppiumSessions () {
+    throw new errors.NotImplementedError('Not implemented yet. ' +
+      'Please check https://github.com/appium/appium/issues/20880 for more details.');
+  }
+
+  async getAppiumSessionCapabilities () {
```

Comment:
this command name should be explicitly added to the umbrella driver commands list

Reply:
like https://github.com/appium/appium/pull/20936/commits/289bbab0237f0c1070028c45059a00c80d01a782?

Reply:
yes